routes/user/user.js

```diff
@@ -132,6 +132,55 @@ module.exports = {
       }
     }
   },
+  updateUser: async (id, user, res) => {
+    try {
+      if (!user) {
+        throw new Error('Missing user request body.');
+      }
+
+      const existingUser = await knex('user')
+        .where({ id: id });
+
+      if((existingUser || []).length == 0) {
+        res.status(400).json({ message: "Invalid user" });
+      } else {
+        if (!user.address) {
+          throw new Error('Missing user address details.');
+        }
+
+        if (!user.address.locality) {
+          throw new Error('Missing user address locality.');
+        }
+
+        const localityId = await knex('locality')
+          .where({ name: user.address.locality })
+          .first('id');
+        
+        if (!localityId || localityId.id) {
```

Comment:
This line should be `if (!localityId || !localityId.id) {`. I made a mistake in writing this line when I wrote the create user API. Could you fix this one line in that API too, and send it in the same check in? I'm just asking because I don't want this same mistake to be made in 2 different places.